cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
Same thing here, what is an example of when we would use this file?

Reply:
yup, we will be using this soon.